/DriveModelNode:
  ros__parameters:
    frequency: 10
    wp_topic: "/waypoints/current"
    odom_topic: "/odom/filtered"
    ackermann_topic: "/control/drive"
    track_width: 3.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false

